torchao/quantization/quant_primitives.py

```diff
@@ -171,7 +180,7 @@ def quantize_affine(
       output_dtype (torch.dtype): requested dtype (e.g. torch.uint8) for output Tensor
       quant_min (Optional[int]): minimum quantized value for output Tensor, if not specified, it will be derived from dtype
       quant_max (Optional[int]): maximum quantized value for output Tensor, if not specified, it will be derived from dtype
-      zero_point_domain (ZeroPointDomain): the domain that zero_point is in, should be either integer or float
+      zero_point_domain (ZeroPointDomain): the domain that zero_point is in, should be eitehr integer or float
```

Comment:
please revert these changes, there is a PR that fixes these typos and seems it's reverted here